src/utils.ts

```diff
@@ -298,3 +301,33 @@ export class UDC {
         return UDC.instance;
     }
 }
+
+export function readContract(contractPath: string) {
+    const parsedContract = parse(
+        fs.readFileSync(contractPath).toString("ascii")
+    ) as CompiledContract;
+    return {
+        ...parsedContract,
+        program: compressProgram(parsedContract.program)
+    };
+}
+export function jsonToBigInt(alwaysParseAsBig: boolean) {
```

Comment:
I think this is a misleading name. Makes it sound like we are converting a json to a bigint (we aren't, right?).

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/293/commits/672522ac4a38bc9e70bdfd80a54a6fd9852da162